oneibl/webclient.py

```diff
@@ -12,6 +12,7 @@
 
 from ibllib.misc import pprint, print_progress
 
+SDSC_ROOT_PATH = Path('/mnt/ibl')
```

Comment:
This should be a parameter, rather than being hard-coded, or better yet, it could be taken from Alyx